src/v2/Apps/Purchase/Components/PurchaseHistory.tsx

```diff
@@ -1,85 +1,101 @@
 import {
   Box,
-  Button,
   Flex,
   Image,
   LargePagination,
   Link,
   Sans,
-  Serif,
+  Separator,
   Spinner,
-  StackableBorderBox,
 } from "@artsy/palette"
+import { DateTime } from "luxon"
 import { PurchaseHistory_me } from "v2/__generated__/PurchaseHistory_me.graphql"
 import React, { useState } from "react"
 import { RelayRefetchProp, createRefetchContainer, graphql } from "react-relay"
 import { get } from "v2/Utils/get"
+import styled from "styled-components"
 
 interface OrderRowProps {
   order: PurchaseHistory_me["orders"]["edges"][number]["node"]
+  hasDivider: boolean
 }
+
+const StyledImage = styled(Image)`
+  object-fit: cover;
+  height: 50px;
+  width: 50px;
+`
+
 const OrderRow = (props: OrderRowProps) => {
   const { order } = props
   const artwork = get(order, o => o.lineItems.edges[0].node.artwork)
 
+  const orderCreatedAt = DateTime.fromISO(order.createdAt).toLocaleString(
+    DateTime.DATE_SHORT
+  )
   if (!artwork) {
     return null
   }
 
+  const orderIsInactive =
+    order.state === "ABANDONED" || order.state === "CANCELED"
+
   return (
-    <Box py={1}>
-      <StackableBorderBox
+    <Box px="20px">
+      <Flex
+        py="15px"
         flexDirection="row"
         alignItems="center"
         justifyContent="space-between"
       >
-        <Flex width="350px">
-          <Flex height="auto" alignItems="center" mr={2}>
-            <Image
-              src={get(artwork, a => a.image.resized.url)}
-              alt={artwork.title}
-              width="55px"
-            />
-          </Flex>
-          <Flex flexDirection="column" justifyContent="center">
+        <Flex
+          alignItems="center"
+          justifyContent="center"
+          height="50px"
+          width="50px"
+          mr="15px"
+        >
+          <StyledImage
+            src={get(artwork, a => a.image.resized.url)}
+            alt={artwork.title}
+          />
+        </Flex>
+        <Flex flexDirection="column" justifyContent="center" width="100%">
+          {!orderIsInactive && (
             <Link
               href={`/orders/${order.internalID}/status`}
               underlineBehavior="hover"
             >
-              <Serif size="2" weight="semibold">
-                {artwork.artist_names}
-              </Serif>
+              <Sans size="3t">{artwork.artist_names}</Sans>
             </Link>
-            <Serif italic size="2" color="black60" lineHeight={1.3}>
-              {artwork.title}, {artwork.date}
-            </Serif>
-          </Flex>
+          )}
+          {orderIsInactive && <Sans size="3t">{artwork.artist_names}</Sans>}
+
+          <Sans size="3t" color="black60" lineHeight={1.3}>
```

Comment:
It's somewhat surprising to see a lineHeight change manually being specified. Doesn't need to change exactly, but I wonder if something like changing from `3t` to `3` would give the desired outcome.

Reply:
Again, not necessarily something that needs to change on this PR.

Reply:
For text we have our `Text` component (which we should be using for all new code): https://palette.artsy.net/tokens/typography. All of the updated values are here, with baked in line-heights.

Reply:
Isn't 3t taller than 3?

Reply:
3t is a smaller line height than 3. It stands for _3 tight_.

@damassi's comment about updating to the `Text` component is on point (thanks Chris). @lilyfromseattle you may need to talk to @willanderson to get a translation of what the updated type style components should be. Give that this isn't released I'm _ok_ with you merging as is, but following up w/ that next will be good. There's a separate initiative to update typography styles to `Text` which is coming up soon and ideally we'd update this _before_ that initiative happens. If you could talk to Will about that today and pass over that functionality that would be great!

I'm happy to pair or answer any questions you have on updating typography.